### PR TITLE
Issue #2188 Update usage message for interactive use of hostfolder add

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

**Comment by praveenkumar** [comment]
retest this please.

**Comment by anjannath** [comment]
This still needs some brain cycles... ideally we should ask for all the info when its interactive mode, Including the type and hostfolder name.
```
$ minishfit hostfolder add --interactive
Type: ...
Source: ...
Target: ...
Name: ...
```

**Comment by praveenkumar** [comment]
@anjannath if you go through the code that is not something we can do because according to your type it ask required information like for sshfs it asks source and target but in case of cifs it ask different info :(

File: cmd/minishift/cmd/hostfolder/add.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ import (
 
 const (
 	usage                = "Usage: minishift hostfolder add --type TYPE --source SOURCE --target TARGET HOST_FOLDER_NAME"
+	interactiveModeUsage = "Usage: minishift hostfolder add --interactive --type TYPE [sshfs|cifs (Default: cifs)] HOST_FOLDER_NAME"
```
Comment (anjannath) [inline cmd/minishift/cmd/hostfolder/add.go]: :+1:

**Comment by anjannath** [comment]
@praveenkumar I see... but it can be done i guess, based on the interactive input of type, we can ask for the type the first thing, and based on that decide what info to ask the user.. anyway this needs to happen on a separate issue, The message looks fine now, better than before.

**Comment by LalatenduMohanty** [comment]
@praveenkumar  I do not think the original issue is getting fixed by this PR. 
```
$ minishift hostfolder add --interactive
Usage: minishift hostfolder add --interactive --type TYPE [sshfs|cifs (Default: cifs)] HOST_FOLDER_NAME
```

I still do not see the prompt asking me to give input for the arguments.

**Comment by LalatenduMohanty** [comment]
> @praveenkumar I do not think the original issue is getting fixed by this PR.

#2219

**Pull request merged**